### PR TITLE
Be consistency for assertion methods call

### PR DESCRIPTION
**Pull request by peter279k**
As title.

### CODE REVIEW

File: test/FeatureTest.php [hunk 1]
```diff
@@ -21,9 +21,9 @@ public function testItShouldBeCreatedWithIdEmptyStrategiesAndStatus(): void
             false
         );
 
-        $this->assertSame(self::FEATURE_ID, $feature->id());
-        $this->assertCount(0, $feature->strategies());
-        $this->assertFalse($feature->isEnabled());
+        self::assertSame(self::FEATURE_ID, $feature->id());
```
Comment (kpicaza) [inline test/FeatureTest.php]: We are using $this in other packages, I think we will do the same and maintain $this for all

**Review by xserrat — APPROVED** [review]
🚀 🚀  Thanks for your contribution :)

File: test/FeatureTest.php (hunk 1 above)
Comment (xserrat) [reply test/FeatureTest.php]: I am agree with @kpicaza . It's true that the `Assert` abstract class from PHPUnit has these methods as `static` so they can be called in this way. But we can leave the method calls using `$this` as part of the `TestCase` instance :)

**Comment by peter279k** [comment]
Done :).

**Pull request merged**